inst/stan/functions/obs_lpmf.stan

```diff
@@ -21,14 +22,25 @@ real obs_lpmf(int[] dummy, int start, int end, int[] obs, int[] sl, int[] csl,
     tar_obs = imp_obs[g][t];
     // allocate report day effects
     rdlh = srdlh[rdlurd[t:(t + l - 1), g]];
+    // allocate share of cases with known reference date
+    tar_alpha = alpha[g][t];
     // combine expected final obs and date effects to get expected obs
     esnap += l;
-    exp_obs[ssnap:esnap] = expected_obs(
+    vector[l] exp_obs_all = expected_obs(
       tar_obs, ref_lh[1:l, dpmfs[i]], rdlh, ref_p
     );
+    // compute expected final obs with known and missing reference date
+    exp_obs[ssnap:esnap] = exp_obs_all * tar_alpha;
+    if(t+l-1>=1+dmax){
+      exp_obs_miss[g][max(1 + dmax, t):(t + l - 1)] += exp_obs_all[max(2 + dmax - t, 1):l] * (1 - tar_alpha);
+    }
```

Comment:
Here we have a potentially serious conflict with the current model setup. The problem is that we currently do not model the `exp_obs` for the maximum delay. While this decision is fine for the cases with known reference date, it is a problem for the cases with unknown reference date, because the exp_obs at the maximum delay are missing in the convolution over reference dates. This should bias the model, although not super strongly if the maximum delay is large enough. But it is a general issue to discuss. For now, I have just implemented the biased version.

Reply:
We had a chat about this and the general conclusion is that I handled the normalisation of the delay distributions incorrectly but in a way that biased them vs recovering from simulation rather than impacting the nowcasts. Here they would also bias the nowcasts so time to change. 

The preferred solution was to lump all the truncated probability onto the maximum delay which mimics the handling of observations in the R code. We also thought it would make sense to do this as standard in the R code with no option to do anything else for now. 

 @adrian-lison if I am not mistaken you said you would take point on this?

Reply:
@seabbs Exactly, got this on my list